src/wfqc/data.py

```diff
@@ -1,53 +1,52 @@
 """
-Functions to download data.
-"""
+Module to download meta data about software in bio.tools
 
-import pandas as pd         
+"""
 import os
 from tqdm import tqdm       
 from datetime import datetime, timedelta
 import glob
 import json
 import numpy as np
-import asyncio              # -"-
-import aiohttp              # Used for aggregating requests into single session
+import asyncio              
+import aiohttp              
 import nest_asyncio         # For jupyter asyncio compatibility 
 nest_asyncio.apply()        # Automatically takes into account how jupyter handles running event loops
-import jsonpath_ng as jp    # TODO: import jsonpath_ng.ext      # More efficient json processing look into if actually computationally more efficient 
 
 # TODO: import jsonpath_ng.ext      # More efficient json processing look into if actually computationally more efficient 
-import requests             # For single API requests 
-import pickle
 
 
-async def aggregate_requests(session, url):
-    """ 
-    Sync the bio.tools (page) requests so they are all made in a single session 
-
-    Parameters
-    ----------
-    session : aiohttp.client.ClientSession object
-        session object for package aiohttp
-    url : str
-        url for request
+async def aggregate_requests(session: aiohttp.ClientSession, url: str) -> dict:
```

Comment:
I see that you moved from `NumPy` style to `reST` style of documentation. Both are fine btw.